task/main.py

```diff
@@ -41,33 +44,34 @@ def __init__(self, event):
         table_resource = boto3.resource('dynamodb', region_name='us-west-2')
         self.db_table = table_resource.Table(os.getenv('table_name', default='DiscoverGranulesLock'))
         # Note: logging.DEBUG and logging.INFO seem to have the same result with the cumulus debugger
-        logging_level = logging.INFO if os.getenv('enable_logging', 'false').lower() == 'true' else logging.WARNING
-        self.logger = CumulusLogger(name='Recursive-Discover-Granules', level=logging_level)
 
     def discover(self):
         """
         Helper function to kick off the entire discover process
         """
         granules = self.collection.get('meta', {}).get('granules', None)
         if granules:
-            self.logger.info(f'Received {len(granules)} to reingest.')
+            logger.info(f'Received {len(granules)} to reingest.')
```

Comment:
Can you add the collection name to the log?

Reply:
https://github.com/ghrcdaac/discover-granules-tf-module/pull/49/files/b4c32407c019b95a31e8259cd419e68c1b37ef27#r770729224